### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by milanzmitrovic**

### CODE REVIEW

File: examples/tooltip.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
+
+from dash import Dash, dcc, html, Input, Output, no_update
+import pandas as pd
+import plotly.express as px
+import plotly.io as pio
+import dash_mantine_components as dmc
+
+
+pio.renderers.default = 'browser'
```
Comment (Coding-with-Adam) [inline examples/tooltip.py:9]:
```suggestion
```

File: examples/tooltip.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
+
+from dash import Dash, dcc, html, Input, Output, no_update
+import pandas as pd
+import plotly.express as px
+import plotly.io as pio
```
Comment (Coding-with-Adam) [inline examples/tooltip.py:5]:
```suggestion
```

File: examples/tooltip.py [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+            html.H4(f"Year-Month: {date_month}"),
+            dcc.Graph(figure=fig1),
+
+        ], style={'width': '200px',
+                  'height': '100px',
+                  'white-space': 'normal'
+                  }
+        )
+    ]
+
```
Comment (Coding-with-Adam) [inline examples/tooltip.py:100]: @milanzmitrovic would you be able to add a few comments inside the callback function that help the user understand a few lines of code that are relatively new for users. For example, what's the purpose of:
- `date_month = x[:7]`
- `fig1 = px.bar(...)`
- the bbox?
- the `children` on line 88

**Review by Coding-with-Adam — CHANGES_REQUESTED** [review]
Thanks @milanzmitrovic Very nice tooltip example app. I had one comment for you above regarding the callback function.

**Comment by milanzmitrovic** [comment]
Yeah, that makes sense. I will update it pretty soon.

**Pull request closed without merging**